### PR TITLE
[SD-2412] - Send/Fetch Workflow

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/features/content_move.feature [hunk 1]
```diff
@@ -86,6 +86,162 @@ Feature: Move or Send Content to another desk
...
+          "desk": "#desks._id#",
+          "incoming_macro": "validate_headline"
+          }
+        ]
+        """
+        And we post to "/archive/123/move"
+        """
+        [{"task": {"desk": "#desks._id#", "stage": "#stages._id#"}}]
+        """
+        Then we get error 412
```
Comment (mdhaman) [inline server/features/content_move.feature]: Error code should be 400 not 412.

File: server/features/content_move.feature [hunk 2]
```diff
@@ -86,6 +86,162 @@ Feature: Move or Send Content to another desk
...
+          "task_status": "in_progress",
+          "desk": "#desks._id#"
+          }
+        ]
+        """
+        And we post to "/archive/123/move"
+        """
+        [{"task": {"desk": "#desks._id#", "stage": "#stages._id#"}}]
+        """
+        Then we get error 412
```
Comment (mdhaman) [inline server/features/content_move.feature]: same as above.

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**